spdmlib/src/requester/get_measurements_req.rs

```diff
@@ -15,28 +15,40 @@ use crate::protocol::*;
 use crate::requester::*;
 
 impl RequesterContext {
+    #[allow(clippy::too_many_arguments)]
     async fn send_receive_spdm_measurement_record(
         &mut self,
         session_id: Option<u32>,
         measurement_attributes: SpdmMeasurementAttributes,
         measurement_operation: SpdmMeasurementOperation,
         spdm_measurement_record_structure: &mut SpdmMeasurementRecordStructure,
+        transcript_vca: &mut Option<ManagedBufferA>,
+        transcript_meas: &mut Option<ManagedBufferM>,
```

Comment:
If the caller needs to invoke send_receive_spdm_measurement_record() multiple times, then transcript_vca will be copied multiple times.

That is not efficient.

Reply:
> If the caller needs to invoke send_receive_spdm_measurement_record() multiple times, then transcript_vca will be copied multiple times.
> 
> That is not efficient.
> 
> Can we do that in init_connection()?

Sure, updated.